lib/Doctrine/CouchDB/CouchDBClient.php

```diff
@@ -565,4 +567,99 @@ public function getRevisionDifference($data)
         }
         return $response->body;
     }
+
+    /**
+     * Transfer missing revisions to the target.
+     *
+     * @param string $docId
+     * @param array $missingRevs
+     * @param CouchDBClient $target
+     * @return array|HTTP\ErrorResponse|string
+     * @throws HTTPException
+     */
+    public function transferChangedDocuments($docId, $missingRevs, CouchDBClient $target)
+    {
+        $path = '/' . $this->getDatabase() . '/' . $docId;
+        $params = array('revs' => true ,'latest' => true,'open_revs' => json_encode($missingRevs));
+        $query = http_build_query($params);
+        $path .= '?' . $query;
+
+        $targetPath = '/' . $target->getDatabase() . '/' . $docId . '?new_edits=false';
+
+        $mutltipartHandler = new MultipartParserAndSender($this->getHttpClient(), $target->getHttpClient());
+        return $mutltipartHandler->request(
+            'GET',
+            $path,
+            $targetPath,
+            null,
+            array('Accept' => 'multipart/mixed')
+        );
+
+    }
+
+    /**
+     * Get changes as a stream.
+     *
+     * This method similar to the getChanges() method. But instead of returning
+     * the set of changes, it returns the connection stream from which the response
+     * can be read line by line. This is useful when you want to continuously get changes
+     * as they occur.
+     *
+     * @param array $params
+     * @param bool $raw
+     * @return resource
+     * @throws HTTPException
+     */
+    public function getChangesAsStream(array $params = array())
+    {
+        // Set feed to continuous.
+        if (!isset($params['feed']) || $params['feed'] != "continuous") {
```

Comment:
I think the code style standard would prefer single quotation marks in places like this.